src/javascript/app/pages/trade/process.js

```diff
@@ -5,6 +5,7 @@ const Contract         = require('./contract');
 const Defaults         = require('./defaults');
 const Durations        = require('./duration');
 const GetTicks         = require('./get_ticks');
+const lookback         = require('./lookback').Lookback;
```

Comment:
since this is a module, it should be capital so `const Lookback         = require('./lookback');`, without the `.Lookback` at the end because it should be a variable not an object if it's a single export